frontend/src/features/public-form/PublicFormProvider.tsx

```diff
@@ -205,6 +206,25 @@ export const PublicFormProvider = ({
         return showErrorToast(error, form)
       }
 
+      const countryRegionFieldIds = new Set(
+        form.form_fields
+          .filter((field) => field.fieldType == BasicField.CountryRegion)
+          .map((field) => field._id),
+      )
+      const formInputsWithCountryRegionInUpperCase = Object.keys(
```

Comment:
It would be helpful if you could add a comment to explain why we are mapping the CountryRegion input to upper case! (for consistency with myinfo countries)

Reply:
Yes, I can. I hope the comment I just added is sufficient (let me know if it sounds confusing).